docs/running/overview.rst

```diff
@@ -2,16 +2,69 @@
 
 Overview
 ========
-This section describes how to run Toil in various environments, including locally, in the cloud, and with using CWL. 
-
-Specifically, the running Toil section documents detail for the following:
-
-.. toctree::
-   cli
-   cwl
-   deploying
-   amazon
-   azure
-   openstack
-   gce
-   hpcEnvironments
+This section describes how to run Toil in various environments, including :ref:`locally <fileJobStore>`,
+:ref:`in the cloud <cloudOverview>`, and using :ref:`CWL <cwl>`.
+
+Toil is built in a modular way so that it can be used on lots of different systems and with different configurations.
+The three configurable pieces are the
+
+ - :ref:`jobStoreOverview`,
+ - :ref:`batchsystemOverview`, and
+ - :ref:`provisionerOverview`.
+
+.. _jobStoreOverview:
+
+Job Store
+---------
+
+The job store is a storage abstraction which contains all of the information used in a Toil run. This includes all
+of the files used by jobs in the workflow and also the details of the progress of the run. If a workflow crashes
+or fails, the job store contains all of the information necessary to resume with minimal repetition of work.
+
+Several different job stores are supported.
+
+.. _fileJobStore:
+
+File Job Store
+~~~~~~~~~~~~~~
+
+The file job store keeps the workflow information in a directory on the machine where the workflow is launched.
+This is the simplest and most convenient job store for testing or for small runs.
+
+For an example that uses the file job store, see :ref:`quickstart`.
+
+Cloud Job Stores
```

Comment:
I'd include how to specify these here.  Something like (I didn't check if these are correct):

```
aws:west-2c:jobstorename
azure:west-2c:jobstorename
gce:west-2c:jobstorename
```

Reply:
I don't think this is a good idea because people may assume the can just copy and paste what's above and it would work. This is definitely not the case and reading the surrounding documentation is important for installation, context, etc.......